website/src/models/group.js

```diff
@@ -164,7 +164,8 @@ GroupSchema.methods.sendChat = function(message, user){
   // Kick off chat notifications in the background.
   var lastSeenUpdate = {$set:{}, $inc:{_v:1}};
   lastSeenUpdate['$set']['newMessages.'+group._id] = {name:group.name,value:true};
-  if (NO_CHAT_NOTIFICATIONS.indexOf(group._id) !== -1 || group.memberCount > 5000) {
+  var LARGE_GROUP_CUTOFF = 5000; // Guilds larger than 5000 cannot receive notifications.
```

Comment:
Lets put this after the requires at the top of the file and make it a `const` instead of a `var`